### DISCUSSION TITLE
Negate of an action

### DISCUSSION THREAD

**Discussion by rkrater** [open-circle/valibot#725] [Q&A]
Is there away to negate any of the actions? I'm trying to see if an email address doesn't ending @example.com

`const EmailSchema = v.pipe(
  v.string(),
  v.trim(),
  v.email(),
  v.not.endsWith('@example.com')
);`

**Comment by fabian-hiller** [comment]
For this use case I recommend using `regex` or `check`. There is currently no `endsNotWith` action but feel free to create one. It is very simple. I am happy to help you create a PR.
